DoLahTest/ParserUnitTest.cpp

```diff
@@ -70,6 +77,31 @@ namespace DoLahTest {
         );
     }
 
+    TEST_METHOD(parseAddDetailedTest5) {
+        parseAddTestMethod((std::string)
+            "#cs2103 #homework on the #stage from 15th to 21st",
+            { "15/10/2015 ~ 21/10/2015", "{ stage, homework, cs2103 }", "#cs2103 #homework on the #stage" }
+        );
+    }
+
+    TEST_METHOD(parseAddDetailedTest6) {
+        parseAddTestMethod((std::string)
+            "workout from 15th of Nov to 21st of Dec",
+            { "15/11/2015 ~ 21/12/2015", "{  }", "workout" }
+        );
+    }
+
+    TEST_METHOD(parseAddDetailedTest7) {
+        parseAddTestMethod((std::string)
+            "workout from 10-18-2015 to 10-25-2015",
+            { "18/10/2015 ~ 25/10/2015", "{  }", "workout" }
+        );
+    }
+
+    TEST_METHOD(getMonthTest1) {
+        Assert::AreEqual((std::string) "1", std::to_string(DoLah::DateTimeParser::getMonth("feb")));
```

Comment:
Yeah I added it because of the abovementioned bug by NULL==0. std::tm month starts from 0.